docs/overloading-wp-behavior.md

```diff
@@ -257,6 +315,12 @@ export default function Page() {
 }
 ```
 
+> [!NOTE]
+> Similar to blocks, you can override template parts in two ways:
+
+-   Globally by defining overrides in `snapwp.config.ts`.
+-   Per route by passing `blockDefinitions` as a prop to `EditorBlocksRenderer`.
```

Comment:
You need to prefix them with `>` if you want them in the call out. 

But also, kinda redundant.

How about something closer to "Template Parts are just another type of Block, which means they can be overwritten `[the same way]( link to blocks section of this doc)` in the existing note at the top of this section?